js/api-augment/definitions/msa.ts

```diff
@@ -100,50 +114,17 @@ export default {
                 type: 'ProviderId',
               },
             ],
-            type: 'Option<Vec<SchemaId>>',
```

Comment:
The version 2 was used in frontend side even-though it was not bumped in the node side. I bumped it to 2 so this version in here will stay the same as 2. I don't think there are any reasons to keep version 1 around but I'm open to counter-arguments to why that might be necessary.

Reply:
Related to my other comment, I think we want to keep v1 around, but here are the only reasons I can think of to have it here:

1. To stop those annoying messages of "api not..." from Polkadotjs
2. To limit the breaking points for upgrades.

`#1` there is actually the most convincing for me 😆

Reply:
I was not able to see that error you mentioned inside Polkadotjs when I ran it. But regardless it seems to be need to keep the implementation as is since runtime doesn't let me to implement multiple versions